spec/lib/task-spec.js

```diff
@@ -39,6 +39,7 @@ describe("Task", function () {
         var Logger = helper.injector.get('Logger');
         Logger.prototype.log = sinon.spy();
         Task = helper.injector.get('Task.Task');
+        _ = helper.injector.get('_');
```

Comment:
Looks like .jshintrc doesn't mark global.helper as a lint exception. 
This could pass locally